apps/laboratory/tests/siwe-email.spec.ts

```diff
@@ -49,14 +51,25 @@ emailSiweTest.afterAll(async () => {
   await page.page.close()
 })
 
-// -- Tests --------------------------------------------------------------------
-emailSiweTest('it should sign', async ({ library }) => {
-  const namespace = library === 'solana' ? 'solana' : 'eip155'
-  await page.sign(namespace)
-  await page.approveSign()
-  await validator.expectAcceptedSign()
+emailSiweTest.afterEach(async ({ browserName, timingRecords }, testInfo) => {
+  if (browserName === 'firefox') {
+    return
+  }
+  await afterEachCanary(testInfo, timingRecords)
```

Comment:
do we need this on other tests ?

ie. solana email, socials, etc?

Reply:
Added to SIWE email since it covers both email & SIWE. Are socials different enough to warrant their own canaries? Same with Solana, would failures happen in Solana only and not regular email?

Reply:
on second thought, no
this should suffice. All we care about is making sure embedded wallet is up and running imo